docs/sma/plugin-pdf.md

```diff
@@ -16,27 +16,42 @@ You can use <dfn aria-label="Selection mode that ignores text or image under the
 
 ### PDF: Built-in OCR
 
-The PDF plugin implements an optional **OCR**, working with both **text** and **scientific formulas** (math, chemestry, ...). Make sure that you also configure the [LaTeX plugin](plugin-LaTeX).
+The PDF plugin implements an optional **OCR** functionality, which reads out text from images as well as converts scientific formulas (math, chemistry, ...) into Latex. For the latter, please make sure that the separate [LaTeX plugin](plugin-LaTeX) is correctly configured.
 
 #### Setup
 
-Behind the scene, SMA uses [MathPix](https://www.mathpix.com/) to recognize images. [Register](https://dashboard.mathpix.com/) an account to unlock the OCR feature. At the day of writing this guide, pricing is free up to 1000 requests.
+Behind the scene, SMA uses [MathPix](https://www.mathpix.com/) to recognize images. [Register](https://dashboard.mathpix.com/) an account to unlock the OCR feature. As of **24.2.2020**, pricing is free up to 1000 requests (1 request = 1 OCR on image/formula). 
 
-Once you are registered, copy your **MathPix App Name** and **Key** and paste them in the [PDF plugin settings](#).
+!> **Example**: You want to OCR math book which holds 2000 pieces of formulas. First 1000 formulas you OCR are free, the other 1000 formulas will cost a total of $4. However, if you OCR the first 1000 formulas in January and second 1000 in February, it will cost you nothing.
 
 <div class="pure-g">
   <div class="pure-u-1-3" style="padding-right: 30px;">
-    <img src="content/images/plugin-pdf/pdf-mathpix-pricing.png" />
+    <img src="content/images/plugin-pdf/pdf-mathpix-usage.png"  />
+  </div>
+  <div class="pure-u-2-3">
+    <img src="content/images/plugin-pdf/pdf-mathpix-pricing.png"width="220" height="240" />
   </div>
+</div>
+
+*You can find amount of requests you made each month in  https://dashboard.mathpix.com*
+
+
+
+
+
+Once you are registered, copy your **MathPix App Name** and **Key** and paste them in the [PDF plugin settings](#).
+
```

Comment:
Minor improvements + example of payment system to those worried of using credit card etc.